src/StrawberryfieldUtilityService.php

```diff
@@ -97,15 +93,22 @@ class StrawberryfieldUtilityService implements StrawberryfieldUtilityServiceInte
   /**
    * StrawberryfieldUtilityService constructor.
    *
-   * @param \Drupal\Core\File\FileSystemInterface $file_system
-   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
-   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
-   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
-   * @param \Drupal\Core\Entity\EntityFieldManagerInterface $entity_field_manager
-   * @param \Drupal\search_api\ParseMode\ParseModePluginManager $parse_mode_manager
+   * @param  \Drupal\Core\File\FileSystemInterface  $file_system
+   * @param  \Drupal\Core\Entity\EntityTypeManagerInterface  $entity_type_manager
+   * @param  \Drupal\Core\Config\ConfigFactoryInterface  $config_factory
+   * @param  \Drupal\Core\Extension\ModuleHandlerInterface  $module_handler
+   * @param  \Drupal\Core\Entity\EntityFieldManagerInterface  $entity_field_manager
+   * @param  \Drupal\search_api\ParseMode\ParseModePluginManager  $parse_mode_manager
    *   The Search API parse Manager
    */
-  public function __construct(FileSystemInterface $file_system, EntityTypeManagerInterface $entity_type_manager, ConfigFactoryInterface $config_factory, ModuleHandlerInterface $module_handler, EntityFieldManagerInterface $entity_field_manager, ParseModePluginManager $parse_mode_manager) {
+  public function __construct(
```

Comment:
@patdunlavey I see a lot of DCS changes here. Some super good (thx) other a bit crazy and overly strict. I will also give this file a second pass after the merge to restore some of lost read-ability (but still keeping it DCS)

Reply:
Sorry about that. I didn't notice until I had already committed and pushed. If you want, I can reverse these.

Reply:
No worries. Php Storm does a lot to conform to the 80 lines limit but DCS does not apply that rule for functions. I can do that